theseus/geometry/tests/point_types_mypy_check.py

```diff
@@ -0,0 +1,17 @@
+import theseus as th
+
+
+def mypy_operations_check():
+
+    for point_cls in [th.Point2, th.Point3]:
```

Comment:
I tested this locally, and `mypy` doesn't like the syntax `z: point_cls`. I get this error:

```bash
error: Variable "theseus.geometry.tests.point_types_mypy_check.point_cls" is not valid as a type  [valid-type]
theseus/geometry/tests/point_types_mypy_check.py:9: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases
```

I suggest to remove the loop and replace all `: point_cls` for with `: th.Point2` (and again for `: th.Point3`). Also, `mypy` doesn't like repeating the variable `z`, so let's use `z1`, `z2`, `z3`, etc.